recipes/Tedlium2/Tokenizer/tedlium2_prepare.py

```diff
@@ -0,0 +1,184 @@
+"""
+Download link: https://lium.univ-lemans.fr/ted-lium2/
+Shucong Zhang 2023
+"""
+
+import os
+import csv
+import logging
+import torch
+import torchaudio
+import multiprocessing
+from speechbrain.dataio.dataio import merge_csvs
+
+logger = logging.getLogger(__name__)
+
+def clip_and_make_csv(sph_file, stm_file, utt_save_folder, csv_save_folder):
+    """
+    This function splits the .sph Ted-talk recording into utterences based on the .stm annotation
+
+    Arguments
+    ---------
+    sph_file : str
+        Path to the sph file containing Ted-talk recording.
+    stm_file : str
+        Path to the stm file containing Ted-talk annotation.
+    save_folder: str
+        The folder stores the clipped individual utterences.
+    enty: list
+        The list stores the csv entry
+    """
```

Comment:
the fn arguments are not matching the docstring args. Please update the docstring accordingly to the fn parameters.